### ISSUE TITLE
Could you release the checkpoints pretrained on Kinetics 400

### ISSUE THREAD

**Issue by qiyue-liang** [YuanGongND/cav-mae#21]
Hi Yuan, in the paper you mentioned that "Specifically, we train the model on Kinetics-400 (K400) dataset and report the top-1 accuracy on Kinetics-Sounds", I'm wondering if it is possible that you could release the checkpoints pretrained on K400 dataset for action recognition tasks. It appears that the repo currently provides checkpoints only for the event classification tasks. Thanks a lot!

**Comment by YuanGongND** [comment]
hi there, 

Thanks for the inquiry. Unfortunately I do not have time to release this checkpoint. It is not just a checkpoint, but also an inference script (add probably also data preprocessing). Before release, I do a cleanup on my research code and run a brief test. There are many experiments in the appendix and I only have time to release codes for experiments in the main manuscript. 

However, I am willing to answer any questions related to that. 

-Yuan
